fe/fe-core/src/main/java/org/apache/doris/catalog/OlapTable.java

```diff
@@ -1851,4 +1840,8 @@ public void checkReplicaAllocation() throws UserException {
             }
         }
     }
+
+    public Boolean getUseLightSchemaChange() {
```

Comment:
Isn't it a property of OlapTable?
Also, naming it as `lightSchemaChangeEanbled()` seems better.